capstone/static/js/search/search-form.vue

```diff
@@ -52,14 +52,10 @@
             </div>
             <div class="col-1">
               <div class="remfield">
-                <button v-if="fields.length > 1"
-                        class="field-button active"
+                <button type="button"
+                        :class="[fields.length > 1 ? 'active' : 'disabled', 'field-button']"
                         @click="removeField(field['name'])">
                 </button>
-                <button v-if="fields.length <= 1"
-                        class="field-button disabled"
-                        disabled>
```

Comment:
I'm not sure that class="disabled" and the html attribute "disabled" are the same. I think you can still hit enter to submit a button with the css-targeting class "disabled". But I haven't checked.